rplugin/python3/gdb/app.py

```diff
@@ -14,17 +14,22 @@ class App:
     def __init__(self, vim, backendStr, proxyCmd, clientCmd):
         self.vim = vim
 
+        # Prepare configuration: keymaps, hooks, parameters etc.
+        self.config = getConfig(vim)
+        self.defineSigns(self.config)
+
         # Create new tab for the debugging view and split horizontally
-        vim.command("tabnew | sp")
+        cmd = "tabnew | setlocal nowinfixwidth | setlocal nowinfixheight | exe \"normal \<c-w>o\" | "
```

Comment:
Is there any benefit of collecting the commands instead of executing them immediately?